ui/src/jobs/components/ContactList.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+import ExternalLink from '../../common/components/ExternalLink';
+import InlineList from '../../common/components/InlineList';
+
+class ContactList extends Component {
+  static renderContact(contact) {
+    const email = contact.get('email');
+    const name = contact.get('name');
+    const renderParanthesis = name && email;
+    return (
+      <>
+        {name && (
+          <>
```

Comment:
reduntant

Reply:
For this, I couldn't add the space without adding the fragment, I guess I'm doing it wrong